src/test/java/org/prgrms/kdt/util/VoucherMapperTest.java

```diff
@@ -0,0 +1,93 @@
+package org.prgrms.kdt.util;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.prgrms.kdt.enums.VoucherType;
+import org.prgrms.kdt.model.FixedAmount;
+import org.prgrms.kdt.model.PercentAmount;
+import org.prgrms.kdt.model.dto.VoucherDTO;
+import org.prgrms.kdt.model.entity.VoucherEntity;
+
+class VoucherMapperTest {
+
+	@ParameterizedTest
+	@DisplayName("VoucherEntity에서 VoucherDTO로 변경하는 메서드 테스트")
+	@MethodSource("voucherListProvider")
+	void toVoucherDTOTest(List<VoucherDTO> expectedVoucherDTOs, List<VoucherEntity> voucherEntities) {
+		//when
+		List<VoucherDTO> voucherDTOS = voucherEntities.stream()
+			.map(VoucherMapper::toVoucherDTO)
+			.collect(Collectors.toList());
+
+		//then
+		Assertions.assertThat(voucherDTOS)
+			.containsExactlyInAnyOrderElementsOf(expectedVoucherDTOs);
+	}
+
+	@ParameterizedTest
+	@DisplayName("VoucherDTO에서 VoucherEntity로 변경하는 메서드 테스트")
+	@MethodSource("voucherListProvider")
+	void toVoucherEntityTest(List<VoucherDTO> voucherDTOs, List<VoucherEntity> expectedVoucherEntities) {
+		//when
+		List<VoucherEntity> voucherEntities = voucherDTOs.stream()
+			.map(VoucherMapper::toVoucherEntity)
+			.collect(Collectors.toList());
+
+		//then
+		Assertions.assertThat(voucherEntities)
+			.containsExactlyInAnyOrderElementsOf(expectedVoucherEntities);
+	}
+
+	private static Stream<Arguments> voucherListProvider() {
+		List<VoucherDTO> voucherDTOS1 = Arrays.asList(
+			new VoucherDTO(5L, new FixedAmount(100), VoucherType.FixedAmountVoucher),
```

Comment:
```suggestion
		List<VoucherDTO> voucherDTOS1 = List.of(
			new VoucherDTO(5L, new FixedAmount(100), VoucherType.FixedAmountVoucher),
```
p3; 
별건 아니지만 List.of로도 가능합니다!